build.gradle

```diff
@@ -281,6 +281,7 @@ configurations.all {
         force "org.apache.bcel:bcel:6.6.0" // This line should be removed once Spotbugs is upgraded to 4.7.4
         force "com.github.luben:zstd-jni:${versions.zstd}"
         force "org.xerial.snappy:snappy-java:1.1.10.1"
+        force 'com.google.guava:guava:32.0.1-jre'
```

Comment:
The `32.1.1-jre` was out already, should we bump it here (and on `main`)?

Reply:
Opened a PR to bump to 32.1.1-jre: https://github.com/opensearch-project/security/pull/2976

Reply:
I will create a separate PR and backports for that bump. Merging this for now to unblock CI.